torch/onnx/symbolic_opset9.py

```diff
@@ -4482,9 +4482,9 @@ def try_mask_to_index(index):
             rank = symbolic_helper._get_tensor_rank(self)
             if rank is None:
                 raise NotImplementedError(
-                    "Unsupported aten::index operator of advanced indexing on tensor of unknown rank, "
-                    + "try turning on shape and type propagate during export: "
-                    + "torch.onnx._export(..., propagate=True)."
+                    "Unsupported aten::index operator of advanced indexing on tensor of unknown rank. "
+                    + "Try turning on shape inference during export: "
+                    + "torch.onnx._export(..., onnx_shape_inference=True)."
                 )
```

Comment:
```suggestion
                    "Unsupported aten::index operator of advanced indexing on tensor of unknown rank. "
                    "Try turning on shape inference during export: "
                    "`torch.onnx._export(..., onnx_shape_inference=True)`."
                )
```

Reply:
do you know why `lintrunner -a` does not fix that?

Reply:
I think that’s syntactically correct.  lintrunner would respect it